### ISSUE TITLE
v0.6 minting issue

### ISSUE THREAD

**Issue by bananenwilly** [peercoin/peercoin#185]
My qt shows the following:
![bildschirmfoto vom 2017-10-10 13-28-22](https://user-images.githubusercontent.com/11148913/31384377-08e43bb8-adbf-11e7-92cb-af8d8b6a64ea.png)

My daemon displays the same transactions like this:

> 
>     {
>         "account" : "",
>         "address" : "1Wh4bh",
>         "category" : "send",
>         "amount" : 0.00000000,
>         "fee" : 0.27000000,
>         "confirmations" : 518,
>         "blockhash" : "ba8f9a202c87954ce0bacacd5447e5da965fcafc7f98a0bbfdf4b14bb264dbdd",
>         "blockindex" : 1,
>         "blocktime" : 1507516000,
>         "txid" : "221cd7ff5228831066dab81f0a6c7008a3035394b0144a76393f3ef0fd5b2d0f",
>         "time" : 1507516000,
>         "timereceived" : 1507516000
>     },
>     {
>         "account" : "",
>         "address" : "1Wh4bh",
>         "category" : "send",
>         "amount" : 0.00000000,
>         "fee" : 0.12000000,
>         "confirmations" : 471,
>         "blockhash" : "8353fb52e40eb7749b1dacec3ce5209682e459b252508d7904fff8cb52b6ace4",
>         "blockindex" : 1,
>         "blocktime" : 1507532033,
>         "txid" : "43caaa5829c116692e3e5bbdc4be0945efbeff8ef807caf6943aaf845905e91c",
>         "time" : 1507532033,
>         "timereceived" : 1507532033
>     },
>     {
>         "account" : "",
>         "address" : "1Wh4bh",
>         "category" : "send",
>         "amount" : 0.00000000,
>         "fee" : 0.02000000,
>         "confirmations" : 408,
>         "blockhash" : "05c0d2c447e21f4ba243e96b2066e5c38fe9ca357855d77c215c6a1fe29200d8",
>         "blockindex" : 1,
>         "blocktime" : 1507554171,
>         "txid" : "bfc4cd9341b3113da61f6b4d029471b6578f2544c8797c1894907e8211ddeb78",
>         "time" : 1507554171,
>         "timereceived" : 1507554171
>     },
>     {
>         "account" : "",
>         "address" : "1Wh4bh",
>         "category" : "send",
>         "amount" : 0.00000000,
>         "fee" : 0.10000000,
>         "confirmations" : 360,
>         "blockhash" : "7b6b7a505de410c961498379cfa9ad58c60874b32cf15791b464065baf6ce62d",
>         "blockindex" : 1,
>         "blocktime" : 1507568937,
>         "txid" : "c58275b89de9ced97ca551f337afe1ef392ddab25443360569a843254fda103f",
>         "time" : 1507568937,
>         "timereceived" : 1507568937
>     }
> 

I'm not sure why that is.

I might have participated in pacli testing with this wallet.

**Comment by hrobeers** [comment]
can you post the raw transaction?
using `getrawtransaction` rpc method

**Comment by bananenwilly** [comment]
```
getrawtransaction c58275b89de9ced97ca551f337afe1ef392ddab25443360569a843254fda103f
0100000029addb5901bde0a3e6ce139c8fc1d8b907161757caa6cd04a37f0acdbeaa7c56b2ac0641fb010000004948304502206653d36aaec059b7e854d24e739091b6ab75ab1c52d280861670b823e05ed8df022100e6b7c6b1c3fb18bf48e371d488236238d0bd71275167402ac95a691d6104f04a01ffffffff03000000000000000000e08aa904000000002321031c457f2cec12c0dedcd7106720363bc85df6d768d3dcf15332b111458546c54cacf0b1a904000000002321031c457f2cec12c0dedcd7106720363bc85df6d768d3dcf15332b111458546c54cac00000000


decoderawtransaction 0100000029addb5901bde0a3e6ce139c8fc1d8b907161757caa6cd04a37f0acdbeaa7c56b2ac0641fb010000004948304502206653d36aaec059b7e854d24e739091b6ab75ab1c52d280861670b823e05ed8df022100e6b7c6b1c3fb18bf48e371d488236238d0bd71275167402ac95a691d6104f04a01ffffffff03000000000000000000e08aa904000000002321031c457f2cec12c0dedcd7106720363bc85df6d768d3dcf15332b111458546c54cacf0b1a904000000002321031c457f2cec12c0dedcd7106720363bc85df6d768d3dcf15332b111458546c54cac00000000
{
    "txid" : "c58275b89de9ced97ca551f337afe1ef392ddab25443360569a843254fda103f",
    "version" : 1,
    "time" : 1507568937,
    "locktime" : 0,
    "vin" : [
        {
            "txid" : "fb4106acb2567caabecd0a7fa304cda6ca57171607b9d8c18f9c13cee6a3e0bd",
            "vout" : 1,
            "scriptSig" : {
                "asm" : "304502206653d36aaec059b7e854d24e739091b6ab75ab1c52d280861670b823e05ed8df022100e6b7c6b1c3fb18bf48e371d488236238d0bd71275167402ac95a691d6104f04a01",
                "hex" : "48304502206653d36aaec059b7e854d24e739091b6ab75ab1c52d280861670b823e05ed8df022100e6b7c6b1c3fb18bf48e371d488236238d0bd71275167402ac95a691d6104f04a01"
            },
            "sequence" : 4294967295
        }
    ],
    "vout" : [
        {
            "value" : 0.00000000,
            "n" : 0,
            "scriptPubKey" : {
                "asm" : "",
                "hex" : "",
                "type" : "nonstandard"
            }
        },
        {
            "value" : 78.22000000,
            "n" : 1,
            "scriptPubKey" : {
                "asm" : "031c457f2cec12c0dedcd7106720363bc85df6d768d3dcf15332b111458546c54c OP_CHECKSIG",
                "hex" : "21031c457f2cec12c0dedcd7106720363bc85df6d768d3dcf15332b111458546c54cac",
                "reqSigs" : 1,
                "type" : "pubkey",
                "addresses" : [
                    "n39fPRyHTeybJNY5DidxBB6bHvdVwSwe9A"
                ]
            }
        },
        {
            "value" : 78.23000000,
            "n" : 2,
            "scriptPubKey" : {
                "asm" : "031c457f2cec12c0dedcd7106720363bc85df6d768d3dcf15332b111458546c54c OP_CHECKSIG",
                "hex" : "21031c457f2cec12c0dedcd7106720363bc85df6d768d3dcf15332b111458546c54cac",
                "reqSigs" : 1,
                "type" : "pubkey",
                "addresses" : [
                    "n39fPRyHTeybJNY5DidxBB6bHvdVwSwe9A"
                ]
            }
        }
    ]
}

```

**Comment by hrobeers** [comment]
It's a display issue in `gettransaction`
These are just standard minting transactions.

We'll keep this issue open to be investigated and fixed later.

**Comment by bananenwilly** [comment]
Tried it with rc 2. Same result. 
Hope you'll fix it in the next RC or final release.

**Comment by hrobeers** [comment]
it's not protocol related and has been an issue in ppcoin 0.5 too.
We will release a 0.6.1 that fixes non-protocol issues. (unless if someone else makes a PR with these fixes).

My time is limited (all spare time, I don't make money on this).
And we want nodes to be able to start signaling 0.6 ASAP.

**Comment by saeveritt** [comment]
Perhaps https://github.com/bitcoin/bitcoin/issues/779 issue is related.

**Comment by hrobeers** [comment]
good find @saeveritt will backport this for v0.6.2

**Comment by hrobeers** [comment]
More investigation revealed only coinstake transactions are allowed (and required) to have empty outputs by the protocol.
More information about the state of the coinstake transaction can indeed be provided.

**Comment by hrobeers** [comment]
fully fixed by #251
